cunumeric/linalg/exception.py

```diff
@@ -14,5 +14,7 @@
 #
 
 
-class LinAlgError(Exception):
-    pass
+from numpy.linalg.linalg import LinAlgError
+
+# class LinAlgError(Exception):
+#    pass
```

Comment:
no need to leave this commented out code in here

Reply:
I removed the comment but the` flake8` error persists.
It can be called as `num.linalg.LinAlgError`. But in `flake8`, it is showed as  'numpy.linalg.linalg.LinAlgError'.
It looks like the fix didn't match the` flake8` check.
Could you give me some suggestions on how to resolve this conflict?

Reply:
Sorry I did not mean to imply those were related, they are separate comments. Docs for suppressing flake8 errors is here:

https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html